uiowa/uiowa
# Attempt to get default content working again

Resolves https://github.com/uiowa/uiowa/issues/5321

Based on https://github.com/uiowa/uiowa/issues/5321#issuecomment-1170227279, https://github.com/uiowa/uiowa/issues/5321#issuecomment-1170408565, I moved forward with repurposing the component showcase to contain the "new homepage" content, taking the opportunity to properly set section and block config along the way. The homepage grid now contains image blocks instead of text areas.

Downside is that the component showcase no longer exists and we still don't know _why_ this was even an issue.

Upside is that the initial bug is fixed, default content has been updated to be lean and mean with little manual tweaks, got rid of a virtual_tour remnant.

# To Test

- Delete the contents of the `uiowa/docroot/sites/default/files`
- `ddev drush @default.local sql-drop && ddev drush @default.local cr`
- `ddev blt drupal:install --site=default`
- `ddev drush @default.local uli`
- The front page should be the /home
- The footer contact block should be present
- Going into to layout builder, you can modify existing media references without encountering the ajax error (new or existing media replacements)
- Content has the proper classes in the wysiwyg, etc.

# ToDo

- Reset percy baseline.
